### PR TITLE
Updated versions for weave, flannel, and calico

### PR DESCRIPTION
**Pull request by oneilcin**
Did a quick cluster up and get pods for both aws and gke.  Will let CI do the rest

### CODE REVIEW

File: ansible/roles/kraken.config/files/config.yaml [hunk 1]
```diff
@@ -60,10 +60,10 @@ definitions:
             version: v1.10.0
             location: quay.io/calico/cni
           calicoNode:
-            version: v2.4.1
+            version: v2.6.2
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/config.yaml]: this should remain at v2.4.1 (the previous version)

File: ansible/roles/kraken.config/files/config.yaml [hunk 2]
```diff
@@ -60,10 +60,10 @@ definitions:
...
             location: quay.io/calico/node
           flannel:
-            version: v0.8.0
+            version: v0.9.1
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/config.yaml]: also leave as it was

File: ansible/roles/kraken.config/files/gke-config.yaml [hunk 3]
```diff
@@ -59,10 +59,10 @@ definitions:
             version: v1.10.0
             location: quay.io/calico/cni
           calicoNode:
-            version: v2.4.1
+            version: v2.6.2
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/gke-config.yaml]: leave as it was

File: ansible/roles/kraken.config/files/gke-config.yaml [hunk 4]
```diff
@@ -59,10 +59,10 @@ definitions:
...
             location: quay.io/calico/node
           flannel:
-            version: v0.8.0
+            version: v0.9.1
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/gke-config.yaml]: leave as it was

File: ansible/roles/kraken.config/files/config.yaml (hunk 1 above)
Comment (oneilcin) [reply ansible/roles/kraken.config/files/config.yaml]: How do you know to leave just this one?

File: ansible/roles/kraken.config/files/gke-config.yaml [hunk 5]
```diff
@@ -102,10 +102,10 @@ definitions:
       options:
         containers:
           weave:
-            version: 1.9.8
+            version: 2.1.1
             location: weaveworks/weave-kube
           weave_npc:
-            version: 1.9.8
+            version: 2.1.1
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/gke-config.yaml]: leave as it was

File: ansible/roles/kraken.config/files/gke-config.yaml [hunk 6]
```diff
@@ -102,10 +102,10 @@ definitions:
       options:
         containers:
           weave:
-            version: 1.9.8
+            version: 2.1.1
```
Comment (guineveresaenger) [inline ansible/roles/kraken.config/files/gke-config.yaml]: leave as it was

Comment (guineveresaenger) [reply ansible/roles/kraken.config/files/gke-config.yaml]: actually, this one's fine to update. For cleanup, this stanza should be removed, as it's the same as the default.

File: ansible/roles/kraken.config/files/gke-config.yaml (hunk 5 above)
Comment (guineveresaenger) [reply ansible/roles/kraken.config/files/gke-config.yaml]: also fine to update.

File: ansible/roles/kraken.config/files/config.yaml [hunk 7]
```diff
@@ -87,10 +87,10 @@ definitions:
       options:
         containers:
           weave:
```
Comment (joejulian) [inline ansible/roles/kraken.config/files/config.yaml]: We have no automated testing for weave. Have you at least manually tested this version?

I'd rather not change it if we don't test it. At least if it's broken and we didn't touch it it's obvious why it's broken. If we change it it should imply it worked at the time we changed it.

Comment (guineveresaenger) [reply ansible/roles/kraken.config/files/config.yaml]: Do we have automated testing for calico?

Comment (oneilcin) [reply ansible/roles/kraken.config/files/config.yaml]: @joejulian Best way to manually test weave is working?
Note: if we are not going to test this, it shouldn't be in our on-call version check list

**Comment by oneilcin** [comment]
Tested with Weave in the config for both AWS and GKE.  All services and pods were healthy.  Created an nginx pod, and then curled to it from a node other than the one it is running on.  All good!

File: ansible/roles/kraken.config/files/config.yaml (hunk 7 above)
Comment (oneilcin) [reply ansible/roles/kraken.config/files/config.yaml]: tested

**Pull request merged**